test/specs/annotation/spec.js

```diff
@@ -6,14 +6,16 @@ require('./../../utils/starter.spec.js').runTests({
         "/annotation/tests/04.visible_foreign_keys.js",
```

Comment:
Instead of adding this one to the annotation spec, let's create a new spec and call it "json-ld". And then you should have two separate specs:

1. The test cases that are based on data and testing the integration of validator with Reference API and annotation.
2. Test cases that are directly calling the validator.

And if you want to store the json-lds in a separate file like you're already doing, they should not be in `data` folder (data folder is only for the table data). So the whole folder structure would be

```
test/
test/
|-- specs
|   |-- json-ld
|   |   |-- conf
|   |   |   |-- google-dataset
|   |   |   |   |-- data                # Data
|   |   |   |   |   `-- TABLE_NAME.json
|   |   |   |   `-- schema.json         # Schema Definition
|   |   |   `-- google-dataset.conf.json
|   |   |-- files
|   |   |   |-- json-ld-w-whatever.json
|   |   |   |-- other-json-ld-file.json
|   |   |-- tests                       
|   |   |   |-- 01.reference-google-dataset.js
|   |   |   `-- 02.json-ld-validator.js
|   |   `-- spec.js
```

Notice how all the filenames in ermrestjs are kebab-case (lower case with dash) and you should follow the same convention. 

And I think you can add more test cases for the `02.json-ld-validator.js` spec. You should for example test your logic of checking types. It might be easier if you add a comment on top of each file and list the scenarios that you're testing and we can discuss if we should add more or not.